recipes/tbtamr/meta.yaml

```diff
@@ -0,0 +1,55 @@
+{% set name = "tbtamr" %}
+{% set version = "0.0.4" %}
+{% set sha256 = "72db1eb835574f02b0d7768e6b04e310bd2229d0f684100004b20a133eecd882" %}
+
+package:
+  name: "{{ name }}"
+  version: "{{ version }}"
+
+source:
+  url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz"
+  sha256: "{{ sha256 }}"
+
+build:
+  noarch: python
+  number: 0
+  entry_points:
+    - tbtamr=tbtamr.tbtamr:main
+  run_exports:
+    - {{ pin_subpackage('tbtamr', max_pin="x.x.x") }}
```

Comment:
Will the patch release break things?
| Case                             | run_exports statement                                               |
| -------------------------------- | ------------------------------------------------------------------- |
| semantic versioning              | `{{ pin_subpackage("myrecipe", max_pin="x") }}`     |
| semantic versioning (0.x.x)      | `{{ pin_subpackage("myrecipe", max_pin="x.x") }}`   |
| known breakage in minor versions | `{{ pin_subpackage("myrecipe", max_pin="x.x") }}` (in such a case, please add a note that shortly mentions your evidence for that) |
| known breakage in patch versions | `{{ pin_subpackage("myrecipe", max_pin="x.x.x") }}` (in such a case, please add a note that shortly mentions your evidence for that) |

Reply:
@thanhleviet thank you .. do i need to make this change or have you made it and then will accept?

Reply:
Either you or I can do it, what max_pin do you want to change then? I'd go for `x.x` :)

Reply:
@thanhleviet done!! Thanks for your help.